lld/wasm/Symbols.h

```diff
@@ -323,6 +330,12 @@ class DefinedData : public DataSymbol {
   uint64_t size = 0;
 };
 
+class SharedData : public DataSymbol {
```

Comment:
Should this class be called `SharedDataSymbol` for consistency with functions and with the base class?

Reply:
Actually it looks like the larger symbol class hierarchy is fairly inconsistent here. so changing just this class wouldn't really fix that.